Octokit.Tests.Integration/Clients/RepositoriesClientTests.cs

```diff
@@ -1756,21 +1741,33 @@ public class TheTransferByIdMethod
         }
 
         [IntegrationTest]
-        public async Task CanTransferUserRepoToOrg()
+        public async Task TransfersFromUserToOrgWithTeams()
         {
+            // FIXME API doesn't add teams when transferring to an organization
             var github = Helper.GetAuthenticatedClient();
             var newRepo = new NewRepository(Helper.MakeNameWithTimestamp("transferred-repo"));
             var newOwner = Helper.Organization;
+
+            Random rand = new Random();
```

Comment:
we dont want to rely on the test org having to have existing teams.  We have a `TeamContext` that can handle creating a team and destroying it again when it goes out of scope.  So you could rewrite this test to something like this:

``` csharp
        [IntegrationTest]
        public async Task TransfersFromUserToOrgWithTeams()
        {
            // FIXME API doesn't add teams when transferring to an organization
            var github = Helper.GetAuthenticatedClient();
            var newRepo = new NewRepository(Helper.MakeNameWithTimestamp("transferred-repo"));
            var newOwner = Helper.Organization;

            using (var teamContext = await github.CreateTeamContext(Helper.Organization, new NewTeam(Helper.MakeNameWithTimestamp("team"))))
            using (var context = await github.CreateRepositoryContext(newRepo))
            {
                var transfer = new RepositoryTransfer(newOwner, new int[] { teamContext.TeamId });
                await github.Repository.Transfer(context.RepositoryOwner, context.RepositoryName, transfer);

                var repoTeams = await github.Repository.GetAllTeams(context.RepositoryId);

                Assert.True(repoTeams.Any(x => x.Id == teamContext.TeamId));
            }
        }
```

Reply:
Fair enough, I didn't think that one through. I'll change this right now.